interchange/LogicalNetlist.capnp

```diff
@@ -58,6 +58,13 @@ struct InstRef {
 annotation instRef(*) :InstRef;
 using InstIdx = UInt32;
 
+struct SiteTypeRef {
+    type  @0 :Ref.ReferenceType = rootValue;
+    field @1 :Text = "siteTypeList";
+}
+annotation siteTypeRef(*) :SiteTypeRef;
+using SiteTypeIdx = UInt32;
+
```

Comment:
This is a `DeviceResources` construct and should probably stay there to keep the schema clean.